esx_service/vmdk_ops.py

```diff
@@ -601,13 +599,7 @@ def removeVMDK(vmdk_path, vol_name=None, vm_name=None, tenant_uuid=None, datasto
     # Check the current volume status
     kv_status_attached, kv_uuid, attach_mode, attached_vm_name = getStatusAttached(vmdk_path)
     if kv_status_attached:
-        ret = handle_stale_attach(vmdk_path, kv_uuid)
-        if ret:
-            if vol_name is None:
-                vol_name = vmdk_utils.get_volname_from_vmdk_path(vmdk_path)
-            logging.info("*** removeVMDK: %s is in use, volume = %s VM = %s VM-uuid = %s (%s)",
-                vmdk_path, vol_name, attached_vm_name, kv_uuid, ret)
-            return err("Failed to remove volume {0}, in use by VM = {1}.".format(vol_name, attached_vm_name))
+        log_attached_volume(vmdk_path, kv_uuid)
```

Comment:
Yes, log_attached_volume() is needed because this isn;t expected in the code paths where its called.